### PR TITLE
Release/v9.0.5

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- Use the predicted sex cases when generating fam files
- Restrict plink analysis to an intersection between target_capture kits for wes/wgs cases 

### How to test:

- Automatic and continuous test suite
- Run test case hardystag (unknown sex) on hasta
- Run test case pickeddoe (wgs/wes mix) on hasta

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Gatk_combinevariantcallsets.pm [hunk 1]
```diff
@@ -329,16 +325,6 @@ sub analysis_gatk_combinevariantcallsets {
                     sample_info_href => $sample_info_href,
                 }
             );
-            set_file_path_to_store(
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Gatk_combinevariantcallsets.pm:332]: I think this one is needed for genotype (MAF) downstream

File: lib/MIP/Recipes/Analysis/Variant_integrity.pm [hunk 2]
```diff
@@ -25,7 +25,7 @@ BEGIN {
     use base qw{ Exporter };
 
     # Set the version for version checking
-    our $VERSION = 1.17;
+    our $VERSION = 1.19;
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Variant_integrity.pm:28]:
```suggestion
    our $VERSION = 1.18;
```

**Review by henrikstranneheim — DISMISSED** [review]
Excellent work Anders! Very nice that you changed the Io file suffix. It made the rest of the recipes less cumbersome. See comment about gatk_combinevariants bcf file.

File: lib/MIP/Recipes/Analysis/Gatk_combinevariantcallsets.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Recipes/Analysis/Gatk_combinevariantcallsets.pm:332]: It looks like it's a copy of the `set_file_path_to_store` above. I can't see the difference between them but I might be missing something?

**Comment by jemten** [comment]
## Hardystag:
MIP predicts female:
image

Chanjo says male:
image
image

Peddy says male:
image

Plink says female:
image


## Pickeddoe
Now restricts analysis to regions defined in target capture bed
image

File: lib/MIP/Recipes/Analysis/Gatk_combinevariantcallsets.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Gatk_combinevariantcallsets.pm:332]: OK, hard to see other parts in a code review sometimes. As long as its in the deliverables then we should be fine

Comment (jemten) [reply lib/MIP/Recipes/Analysis/Gatk_combinevariantcallsets.pm:332]: I'll take a manual look at the deliverables file to make sure

**Review by henrikstranneheim — DISMISSED** [review]
👍

**Review by henrikstranneheim — DISMISSED** [review]
👍

**Comment by jemten** [comment]
@henrikstranneheim, Chromgraph has been tested and the branch is ready to be merged and deployed

**Comment by jemten** [comment]
Deployed on stage: 
image

Deployed on prod:
image

**Pull request merged**